testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -1128,7 +1128,7 @@ public void testCreateSchema()
             return;
         }
         assertThat(computeActual("SHOW SCHEMAS").getOnlyColumnAsSet()).doesNotContain(schemaName);
-        assertUpdate("CREATE SCHEMA " + schemaName);
```

Comment:
would that work, if execued, and create schema with wrong location, or would that fail?

Reply:
Fails as by default it tries to create data in HDFS.